docs/writing_features_columns.csv

```diff
@@ -1,160 +1,160 @@
-Column Name,Description,Response Options (if applicable),,,,
-essay_id_num,Each writing assignment upload has a unique identifier.  The characters before the underdash matches the unique participant identifier and the characters after the underdash signals the upload number.,N/A,,,,
-Student_ID,Each participant has a unique identifier.,N/A,,,,
+Column Name,Description,Response Options (if applicable)
+essay_id_num,Each writing assignment upload has a unique identifier.  The characters before the underdash matches the unique participant identifier and the characters after the underdash signals the upload number.,N/A
+Student_ID,Each participant has a unique identifier.,N/A
 course_discipline,Academic subject,"? Biology
 ? Business
 ? Computer Science
 ? Criminology 
 ? English
 ? History
-? Sociology",,,,
```

Comment:
Hmm, I am a little concerned about these commas being deleted. Do we no longer have the extra columns?

Reply:
Hi @desilinguist! I will admit that I don't actually see those commas in the penultimate version of this file, so I'm not sure what change is being detected here.  Any thoughts on what it might be?  I only intended to make edits to the last two rows of this file (rows 121 and 122).  Do you want me to create a new pull request just for the writing_features_columns.xlsx file and make the intended edits again?
![image](https://user-images.githubusercontent.com/29762455/94473759-a2a13f80-019a-11eb-99c0-146cfe6b25a6.png)

Reply:
Hmm, I am not sure. Let me try to read in the file in `pandas` and see if it works without the commas.

Reply:
@hmolloy just to confirm there are supposed to be 121 columns in the new version of the CSV file correct?

Also, I think instead of using strings like "Not applicable or not available" for the `Persuasive_Subgenre_Description` column, we should use something like "N/A" since `pandas` will recognize that as not a number rather than a custom string. If you need to explain what "N/A" means, you can do so in the PDF.

Reply:
@desilinguist there are supposed to be 121 columns in the `writing_features.csv` file and 3 columns in the writing_features_columns.csv file (plus 122 rows).

I'm fine with changing `Not applicable or not available` to "N/A" for the `Persuasive_Subgenre_Description` column.  Shall I submit a new pull request?

Reply:
Hmm, when I try to load `writing_feature_columns.csv` , I get a unicode error. 

```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x96 in position 232: invalid start byte
```

Please make sure that you are setting the encoding to UTF-8 when exporting the CSV file.

As for the "N/A" stuff, you can just fix them in this PR itself.

Reply:
Hi @desilinguist.  I've made the following changes and committed them to the `Persuasive subgenre annotations` branch, but I'm not sure if those edits are still associated with this pull request.   

1. I saved those two .csv files with UTF-8 encoding and re-uploaded them.
2. I also changed `Not applicable or not available` to "`N/A`" for the `Persuasive_Subgenre_Description` column.

Please let me know how to proceed.

Thanks so much!